### DISCUSSION TITLE
Unable to connect Telescope

### DISCUSSION THREAD

**Discussion by DSBRobotics** [Stellarium/stellarium#2788] [General]
I have used Stellarium and APT for two years but had to reset computer back to beginning. Using Latest build and Win 11Pro. Attempting to connect scope and cannot get passed Connect button choosing any type of scope even Simulator or with Stellarium or Ascom. Any ideas greatly appreciated.

**Comment by gzotti** [comment]
See #2713.

**Reply by DSBRobotics** [reply]
Thanks so much. Did fix issue.
